### PR TITLE
Additional Updates to CharacterCreation.cs #1

### PR DESCRIPTION
**Pull request by Arthrutus**
Additional OSI-accurate additions to starting templates for elves, humans, and gargoyles. Changes focus on proper armor/weapon additions when making a new character. Gargoyle code
is commented out due to inadequate implementation of the gargoyle race.

### CODE REVIEW

File: Projects/UOContent/Misc/CharacterCreation.cs [hunk 1]
```diff
@@ -991,15 +991,30 @@ private static void SetSkills(Mobile m, SkillNameValue[] skills, int prof)
 
             var addSkillItems = true;
             var elf = m.Race == Race.Elf;
+            var gargoyle = m.Race == Race.Gargoyle;
```
Comment (Arthrutus) [inline Projects/UOContent/Misc/CharacterCreation.cs]: May need to comment this out for now. Could be causing server stability issues.

File: Projects/UOContent/Misc/CharacterCreation.cs [hunk 2]
```diff
@@ -991,15 +991,30 @@ private static void SetSkills(Mobile m, SkillNameValue[] skills, int prof)
...
+                            EquipItem(new HideChest());
+                            EquipItem(new HidePauldrons());
+                            EquipItem(new HideGloves());
+                            EquipItem(new HidePants());
+                            EquipItem(new ElvenBoots());
                         }
+                        //elif (gargoyle)
+                        //{
+                            //EquipItem(new DreadSword());
+                            //EquipItem(new GargishLeatherChestType1());
```
Comment (kamronbatman) [inline Projects/UOContent/Misc/CharacterCreation.cs:1013]: The gargish leather exists.

File: Projects/UOContent/Misc/CharacterCreation.cs [hunk 3]
```diff
@@ -991,15 +991,30 @@ private static void SetSkills(Mobile m, SkillNameValue[] skills, int prof)
...
-                            EquipItem(new LeafChest());
+                            EquipItem(new Circlet());
+                            EquipItem(new HideGorget());
+                            EquipItem(new HideChest());
+                            EquipItem(new HidePauldrons());
+                            EquipItem(new HideGloves());
+                            EquipItem(new HidePants());
+                            EquipItem(new ElvenBoots());
                         }
+                        //elif (gargoyle)
```
Comment (kamronbatman) [inline Projects/UOContent/Misc/CharacterCreation.cs:1010]: Change this to `else if`

File: Projects/UOContent/Misc/CharacterCreation.cs [hunk 4]
```diff
@@ -1010,7 +1025,7 @@ private static void SetSkills(Mobile m, SkillNameValue[] skills, int prof)
                             EquipItem(new StuddedLegs());
                             EquipItem(new ThighBoots());
                         }
-
+
```
Comment (kamronbatman) [inline Projects/UOContent/Misc/CharacterCreation.cs:1028]: Remove this. Probably need to set the IDE to delete extra white spaces on save.

**Comment by kamronbatman** [comment]
I am going to add in the missing gargish items then we can update this PR.

**Review by Arthrutus — COMMENTED** [review]
Fixes made as suggested.

File: Projects/UOContent/Misc/CharacterCreation.cs [hunk 5]
```diff
@@ -1106,19 +1137,31 @@ private static void SetSkills(Mobile m, SkillNameValue[] skills, int prof)
...
+                            EquipItem(new Bokuto());
                         }
+                        else if (gargoyle)
+                        {
+                            //EquipItem(new GargishTalwar()); // IMPLEMENTION NEEDED
+                            EquipItem(new GargishLeatherChestType1());
+                            EquipItem(new GargishLeatherArmsType1());
+                            EquipItem(new GargishLeatherKiltType1()); 
+                            EquipItem(new GargishLeatherLegsType1());
                         else
```
Comment (kamronbatman) [inline Projects/UOContent/Misc/CharacterCreation.cs:1157]: Missing a `}` before else

**Pull request merged**